lib/rules/sotd/pp.js

```diff
@@ -48,7 +43,7 @@ function buildWanted (groups, sr) {
                 wanted += " This Amended Recommendation was produced by incorporating errata after that group closed\\.";
         }
     }
-    wanted = '^' + wanted + '$';
+    wanted = wanted ? '^' + wanted + '$' : '';
```

Comment:
That rule becomes useless for IG Notes, so I think we should remove it from the "IG Note" profile and `wanted` should never be empty.